### PR TITLE
editorial review of lock renewal sample

### PR DESCRIPTION
**Pull request by adamralph**
relates to https://github.com/Particular/docs.particular.net/pull/5227

### CODE REVIEW

File: samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs [hunk 1]
```diff
@@ -61,26 +61,30 @@ private static async Task OverrideQueueLockDuration(string queuePath, string con
...
     #region override-transaction-manager-timeout-net-framework
 
-    static void ConfigureTransactionTimeoutNetFramework(TimeSpan value)
+    static void ConfigureTransactionTimeoutNetFramework(TimeSpan timeout)
     {
         SetTransactionManagerField("_cachedMaxTimeout", true);
-        SetTransactionManagerField("_maximumTimeout", value);
+        SetTransactionManagerField("_maximumTimeout", timeout);
+
+        void SetTransactionManagerField(string fieldName, object value) =>
```
Comment (adamralph) [inline samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs:84]: Previously this method only showed under the .NET Core section, which appears _after_ the .NET Framework section, and that left me wondering how the snippet could work.

File: samples/azure-service-bus-netstandard/lock-renewal/sample.md [hunk 2]
```diff
@@ -17,40 +17,40 @@ include: asb-connectionstring-xplat
...
 
 snippet: native-message-access
 
-The call to extend the lock must be using the same Azure Service Bus connection object and the queue path used to receive the incoming message. The transport exposes this information via `TransportTransaction` and can be accessed in the following manner:
+The request to renew the lock must use the same Azure Service Bus connection object and queue path used to receive the incoming message. These items are available in the `TransportTransaction`:
 
 snippet: get-connection-and-path
 
-With connection object and queue path, an Azure ServiceBus can be created to renew the lock using the message's lock token obtained earlier. This is done in a background task, running in an infinite loop until the cancellation token passed in is signaled as canceled.
+With the lock token, connection object, and queue path, an Azure ServiceBus may be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signaled as canceled.
```
Comment (adamralph) [inline samples/azure-service-bus-netstandard/lock-renewal/sample.md:53]: What is "an Azure ServiceBus" in this context? Will the reader understand this?

File: samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs [hunk 3]
```diff
@@ -61,26 +61,30 @@ private static async Task OverrideQueueLockDuration(string queuePath, string con
...
         SetTransactionManagerField("s_cachedMaxTimeout", true);
-        SetTransactionManagerField("s_maximumTimeout", value);
-    }
+        SetTransactionManagerField("s_maximumTimeout", timeout);
 
-    static void SetTransactionManagerField(string fieldName, object value)
-    {
-        var cacheField = typeof(TransactionManager).GetField(fieldName, BindingFlags.NonPublic | BindingFlags.Static);
-        cacheField.SetValue(null, value);
+        void SetTransactionManagerField(string fieldName, object value) =>
```
Comment (SeanFeldman) [inline samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs:69]: Was `static` removed accidentally?

File: samples/azure-service-bus-netstandard/lock-renewal/ASBS_2/LockRenewal/Program.cs [hunk 4]
```diff
@@ -61,26 +61,30 @@ private static async Task OverrideQueueLockDuration(string queuePath, string con
...
         SetTransactionManagerField("s_cachedMaxTimeout", true);
-        SetTransactionManagerField("s_maximumTimeout", value);
-    }
+        SetTransactionManagerField("s_maximumTimeout", timeout);
 
-    static void SetTransactionManagerField(string fieldName, object value)
-    {
-        var cacheField = typeof(TransactionManager).GetField(fieldName, BindingFlags.NonPublic | BindingFlags.Static);
-        cacheField.SetValue(null, value);
+        void SetTransactionManagerField(string fieldName, object value) =>
```
Comment (SeanFeldman) [inline samples/azure-service-bus-netstandard/lock-renewal/ASBS_2/LockRenewal/Program.cs:69]: Same question about the `static` keyword.

File: samples/azure-service-bus-netstandard/lock-renewal/sample.md (hunk 2 above)
Comment (SeanFeldman) [reply samples/azure-service-bus-netstandard/lock-renewal/sample.md:53]: No, that's off. What it should be is
```suggestion
With the lock token, connection object, and queue path, an Azure Service Bus `MessageReceiver` object can be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signalled as cancelled.
```

**Review by SeanFeldman — COMMENTED** [review]
@adamralph, a comment and a suggestion linked to the question you had. Thanks.

File: samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs (hunk 3 above)
Comment (adamralph) [reply samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs:69]: image

File: samples/azure-service-bus-netstandard/lock-renewal/sample.md (hunk 2 above)
Comment (adamralph) [reply samples/azure-service-bus-netstandard/lock-renewal/sample.md:53]: 👍 btw the US English forms are "signaled" and "canceled".

```suggestion
With the lock token, connection object, and queue path, an Azure Service Bus `MessageReceiver` object may be created to renew the lock. This is done in a background task, running in an infinite loop until the specified cancellation token is signaled as canceled.
```

File: samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs (hunk 3 above)
Comment (SeanFeldman) [reply samples/azure-service-bus-netstandard/lock-renewal/ASBS_1/LockRenewal/Program.cs:69]: Sigh

**Review by SeanFeldman — APPROVED** [review]

**Pull request merged**